cluster/gce/gci/configure-helper.sh

```diff
@@ -1739,7 +1739,11 @@ function start-kube-addons {
   local -r dst_dir="/etc/kubernetes/addons"
 
   # prep addition kube-up specific rbac objects
-  setup-addon-manifests "addons" "rbac"
+  setup-addon-manifests "addons" "rbac/kubelet-api-auth"
+  setup-addon-manifests "addons" "rbac/kubelet-cert-rotation"
+  if [[ "${REGISTER_MASTER_KUBELET:-false}" == "true" ]]; then
```

Comment:
How will this play with upgrades? I.e. what if the cluster has an old node running?

Reply:
Now we preserve the binding on upgrade clusters now and create an empty binding on new clusters.